lms/djangoapps/mobile_api/admin.py

```diff
@@ -3,7 +3,21 @@
 """
 
 from django.contrib import admin
-from config_models.admin import ConfigurationModelAdmin
-from mobile_api.models import MobileApiConfig
+from config_models.admin import ConfigurationModelAdmin, KeyedConfigurationModelAdmin
+from mobile_api.models import MobileApiConfig, AppVersionConfig
 
 admin.site.register(MobileApiConfig, ConfigurationModelAdmin)
+
+
+class AppVersionConfigAdmin(KeyedConfigurationModelAdmin):
+    """ Admin class for AppVersionConfig model """
+    fields = ('platform', 'version', 'expire_at', 'enabled')
+    list_filter = ['platform']
+
+    class Meta:  # pylint: disable=old-style-class
```

Comment:
Better to follow pylint suggestions than to suppress them: http://stackoverflow.com/questions/15257534/django-forms-modelform-pylint-and-new-old-style-classes#comment33517291_19072577